src/main/java/frc/robot/Parameters.java

```diff
@@ -340,15 +346,54 @@
 
     public static final class climber{
             //motor speeds for climbers
-                public static final double CLIMBMOTOR1_SPEED = 0.0;
-                public static final double CLIMBMOTOR2_SPEED = 0.0;
+            public static final double FRONT_MOTOR_SPEED = 0.0;
+            public static final double BACK_MOTOR_SPEED = 0.0;
+
+                public static final class can{
+                        public static final int FRONT_MOTOR_ID = 13;
+                        public static final int BACK_MOTOR_ID = 14;
+                        public static final int BACK_CANCODER_ID = 16;
+                        public static final int FRONT_CANCODER_ID = 17;
```

Comment:
Why are you creating CANCoders here? We only use CANCoders for the swerve modules. We're going to use NEOs on the climber, so you should be interfacing with the encoder from the SparkMax object of the climber motors (using a positional conversion factor, see SwerveModule.java for an example)